examples/ibm-cloudant/README.md

```diff
@@ -0,0 +1,114 @@
+# Example for CloudantV1
+
+This example illustrates how to use the CloudantV1
+
+These types of resources are supported:
+
+* cloudant
+
+## Usage
+
+To run this example you need to execute:
+
+```bash
+$ terraform init
+$ terraform plan
+$ terraform apply
```

Comment:
> The `terraform apply` command executes the actions proposed in a Terraform plan.
The most straightforward way to use `terraform apply` is to run it without any arguments at all, in which case it will automatically create a new execution plan (as if you had run `terraform plan`) and then prompt you to approve that plan, before taking the indicated actions.

So `apply` is implicit with `plan`, however
> The `terraform init` command is used to initialize a working directory containing Terraform configuration files.

So that should be done at least once, but sometimes `terraform plan` requested me to do that, e.g. when I was using the `activity_tracker_instance` module.